llmfoundry/callbacks/async_eval_callback.py

```diff
@@ -194,8 +216,10 @@ class AsyncEval(Callback):
         training_config: Dict[str, Any]: The config from the training run
         interval: Union[str, int, Time]: The interval describing how often eval runs should be
             launched. If an integer, it will be assumed to be in :attr:`.TimeUnit.EPOCH`.
-            Otherwise, the unit must be either :attr:`.TimeUnit.EPOCH`, :attr:`.TimeUnit.BATCH`,
-            :attr:`.TimeUnit.TOKEN`, or :attr:`.TimeUnit.SAMPLE`.
+            Otherwise, the unit must be either :attr:`.TimeUnit.EPOCH` or :attr:`.TimeUnit.BATCH`.
+        check_interval: Optional[Union[str, int, Time]]: The interval describing how often
```

Comment:
does this actually need to be configurable? or we can we just remove this knob?

Reply:
fair, removed!